pkg/agent/multicast/mcast_controller.go

```diff
@@ -131,7 +131,7 @@ func (c *Controller) updateGroupMemberStatus(obj interface{}, e *mcastGroupEvent
 	newStatus := &GroupMemberStatus{
 		group:          status.group,
 		localMembers:   make(map[string]time.Time),
-		remoteMembers:  status.remoteMembers,
+		remoteMembers:  sets.NewString(status.remoteMembers.List()...),
```

Comment:
```suggestion
		remoteMembers: status.remoteMembers.Union(),
```
`List` performs unnecessary sorting.

Reply:
updated.